CONTRIBUTING.md

```diff
@@ -123,6 +119,63 @@ use a shared working branch strategy.
 5. Resolve any merge conflicts.
 6. When your work is complete, merge the shared working branch into `develop` via a PR.
 
+### Working with docs-website-private
+
+The [docs-website-private repository](https://github.com/newrelic/docs-website-private) is a private copy of the docs-website repository. It is meant for private colloboration, on work that is not ready for public consumption.
+
+If you need to interact with the public & private repositories, there are three setups that you can use:
+1. cloning docs-website, adding a remote to the private repository.
+2. cloning docs-website-private, adding a remote to the public repository.
+3. doing both of the above.
+
+Our recommendation is setup #1, but you can use #2 or #3 to do the same things.
```

Comment:
Do we need to provide 3 options here?. Also what is the value in taking the second and third approaches? Should we articulate when you may opt to do 2nd, or 3rd option?